tests/HttpAdapterTest.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
-namespace Http\Adapter\Tests;
+namespace Http\Adapter\Guzzle6\Tests;
 
 use GuzzleHttp\Client;
-use Http\Adapter\Guzzle6HttpAdapter;
+use Http\Adapter\Guzzle6\Client as Adapter;
```

Comment:
Hm, let’s not forget that `Http\Adapter\Guzzle6\Adapter` looks very much redundant. There’s still the option `Http\Adapter\Guzzle6\Guzzle6Adapter|Client` of course.